File: Java/PasscodeGenerator.java
public class PasscodeGenerator{
	public static void main(String[]args) throws java.io.IOException{
		java.io.File file = new java.io.File("passcodes.txt");
		java.io.Writer writer = new java.io.FileWriter(file);
		for (int w = 0; w < 10; w++){
			for (int x = 0; x < 10; x++){
				for (int y = 0; y < 10; y++){
					for (int z = 0; z < 10; z++){
						writer.write(w + "" + x + "" + y + "" + z + "\n");
					}
				}
			}
		}
		writer.flush();
		writer.close();
	}
}

File: Python/PasscodeGenerator.py
#!/usr/bin/env python
# Generates all the possible passcodes of an iPhone device
# With a four-digit passcode
def main():
	keys = open("pass.txt", "w")
	for w in range(0, 10):
		for x in range(0, 10):
			for y in range(0, 10):
				for z in range(0, 10):
					keys.write(str(w) + str(x) + str(y) + str(z) + "\n")
	keys.close()

main()
